src/js/graphics.js

```diff
@@ -29,6 +29,26 @@ function createTextSprite(name, text, colors, scale) {
     return canvas;
 }
 
+// Create and return a custom instructions sprite canvas
+function createJsonSprite(name, vector) {
+    var canvas = makeSpriteCanvas(name);
+    var context = canvas.getContext('2d');
+    if (vector.w) canvas.width *= vector.w;
+    if (vector.h) canvas.height *= vector.h;
+    var json = JSON.parse("[" + vector.data + "]");
```

Comment:
JSON.parse throws a SyntaxError exception when it can't parse the input string

Reply:
So wrap it in a try-catch block, return null.